pilot/pkg/model/context.go

```diff
@@ -473,7 +473,11 @@ func (s *NodeMetaProxyConfig) MarshalJSON() ([]byte, error) {
 
 func (s *NodeMetaProxyConfig) UnmarshalJSON(data []byte) error {
 	pc := (*meshconfig.ProxyConfig)(s)
-	return jsonpb.Unmarshal(bytes.NewReader(data), pc)
+	// TODO(ramaraochavali): Remove this in 1.18. This is temporarily enabled to allow old
+	// proxies connect to Istiod with removed "ParentShutdownDuration".
```

Comment:
we should probably keep this forever for safety?

Reply:
We do that for Istio CRDs

Reply:
Sure